tests/sqllogictests/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+authors = ["Databend Authors <opensource@datafuselabs.com>"]
+edition = "2021"
+license = "Apache-2.0"
+name = "sqllogictests"
+publish = false
+version = "0.1.0"
+
+[dependencies]
+common-exception = { path = "../../src/common/exception" }
+tokio = { version = "1.21.1", features = ["full"] }
+sqllogictest = "0.9.0"
+async-trait = "0.1.53"
+futures-util = "0.3.25"
+mysql = "23.0.0"
```

Comment:
mysql_async

Reply:
I have tried it but met some blocks, so I chose the crate.

For tests, I think the two crates don't have too much difference.  Any thoughts? @PsiACE

Reply:
The upstream seems to fix the bug,  I'll try again. https://github.com/blackbeam/mysql_async/releases/tag/v0.31.2